tests/ArrayHelper/ArrayHelperTest.php

```diff
@@ -2,13 +2,17 @@
 
 declare(strict_types=1);
 
-namespace Yiisoft\Arrays\Tests;
+namespace Yiisoft\Arrays\Tests\ArrayHelper;
```

Comment:
Can we move these just `Helper` directory?

Reply:
You mean rename `Yiisoft\Arrays\Tests\ArrayHelper` to `Yiisoft\Arrays\Tests\Helper` ?

Reply:
Yes.

Reply:
I think class name `ArrayHelper` and folder name with his tests should have equal names.